Gruntfile.js

```diff
@@ -176,25 +176,11 @@ grunt.initConfig( {
 			}
 		}
 	},
-
-	jscs: {
-		ui: {
-			options: {
-				config: true
-			},
-			files: {
-				src: [ "demos/**/*.js", "build/**/*.js", "ui/**/*.js" ]
-			}
+	eslint:{
+		options: {
+			configFile: "eslint.json"
 		},
-		tests: {
-			options: {
-				config: true,
-				maximumLineLength: null
-			},
-			files: {
-				src: [ "tests/**/*.js" ]
-			}
-		}
+		target:["demos/**/*.js", "build/**/*.js", "ui/**/*.js"]
```

Comment:
This doesn't follow the jQuery code style with regards to spacing; doesn't ESLint verify it?